__tests__/matchers.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'vitest';
+import { map } from 'unist-util-map';
+
+const removePosition = ({ position, ...node }) => node;
+
+function toStrictEqualExceptPosition(received, expected) {
```

Comment:
would rec typing the return on this to vitests expectations:

```suggestion
import type { ExpectationResult } from '@vitest/expect';

function toStrictEqualExceptPosition(received, expected): ExpectationResult {
```